app/goal_routes.py

```diff
@@ -0,0 +1,160 @@
+from app import db
```

Comment:
I enjoy that you experimented with making two different routes files!

Reply:
one route file was becoming too busy so I segregated them for easy reference